ucam-department-of-psychiatry/camcops
# Update Black, remove shebangs and noqas

I've updated the pre-commit hook to:

* Use latest stable version of Black
* Check that files with shebangs are executable
* Remove redundant noqas

and fixed any errors.
